Automattic/themes
# Varia themes: fix cover block + layout grid bug

This PR addresses #2976. 

To describe what's going on in more detail: 

- The cover block's default CSS contains `height: 100%;`
- When you place a cover block inside a layout grid column, the cover takes up 100% of the calculated height of the layout grid column, pushing whatever else inside that same column _beyond_ the height the column takes up in the document flow. This is why the footer credit appears in the middle as reported in 2976. 
- It happens to be that the Doyle layout uses this pattern, but any theme that implements the pattern would experience this issue.

This PR targets all direct descendants of layout grid columns to set their `height: auto`. It feels like a bandaid, and I'm not attached to this solution but wanted to put it out there to get a fix in place.

**To test**
- Follow the steps to recreate the pattern described in #2976. You can use any varia theme on a free plan.
- Verify that any elements that are placed after the group block appear to overlap the layout grid
- Apply this patch (D55158), and verify that the issue is resolved
